config/configmap/inferenceservice.yaml

```diff
@@ -31,15 +31,15 @@ data:
             "supportedFrameworks": [
               "sklearn"
             ],
-            "multiModelServer": "true"
+            "multiModelServer": true
```

Comment:
@yuzliu should we default these to false when making the release and have people turn on manually if they want to try the experimental feature ?

Reply:
I think that would be a good idea. We can also update the docs to include that as part of the how to start.